pkg/probes/curl/curl_json.go

```diff
@@ -26,6 +26,9 @@ type Probe struct{}
 //go:embed userdata-template.yaml
 var userDataTemplate string
 
+//go:embed startup-script.sh
+var startupScript string
```

Comment:
```suggestion
//go:embed systemd-template.sh
var systemdTemplate string
```
Renaming to make more consistent with userdata-template.yaml. More broadly, I'd like to rebrand your approach as the "systemd curl probe" vs. the "GCP curl probe" so that things don't get confusing if/when we try to use systemd on AWS in the future